src/plantuml_shell_backend.rs

```diff
@@ -4,21 +4,21 @@ use std::path::PathBuf;
 use std::process::Command;
 
 use crate::plantuml_backend::PlantUMLBackend;
-use failure::Error;
+use anyhow::{bail, Result};
```

Comment:
I cannot spot the usage of `bail`. Is this really needed here ?

Reply:
It's not in the diff because it's a drop-in replacement for the existing usage of `bail`. 

It's not strictly required. Off the top of my head I think it's roughly equivalent to `Err(anyhow::Error::from(...))`